utils/frontend_utils.py

```
@@ -264,14 +269,130 @@ def find_model_directory(model_name: str):
     # 如果都不存在，返回static默认路径
     return (static_model_dir, '/static')
 
+def find_workshop_item_by_id(item_id: str) -> tuple:
+    """
+    根据物品ID查找Steam创意工坊物品文件夹
+    
+    Args:
+        item_id: Steam创意工坊物品ID
+        
+    Returns:
+        (物品路径, URL前缀) 元组，即使找不到也会返回默认值
```

Ruff (0.14.5)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)